tests/test_consistency.py

```diff
@@ -574,3 +574,38 @@ def test_concat_dfsu3d_single_timesteps_generic_vs_dataset(tmp_path):
     assert ds3.end_time == ds4.end_time
     assert ds3.n_items == ds4.n_items
     assert ds3.n_timesteps == ds4.n_timesteps
+
+
+def test_keepdims_removes_singleton_dimension() -> None:
```

Comment:
Great tests ❤️

And I guess it is not relevant to add one with dfs0 or dfs3? (what happens with the keepdims argument then)

Reply:
I suppose dfs3 could be relevant. Let's revisit it when we improve dfs2 functionality.